spec/requests/api/v2/providers/suggest_provider_spec.rb

```diff
@@ -3,49 +3,46 @@
 describe "GET /suggest" do
   let(:jsonapi_renderer) { JSONAPI::Serializable::Renderer.new }
   let(:organisation) { create(:organisation) }
-  let(:provider1) { create(:provider, provider_name: "PROVIDER 1", organisations: [organisation]) }
-  let(:provider2) { create(:provider, provider_name: "PROVIDER 2", organisations: [organisation]) }
   let(:next_recruitment_cycle) { find_or_create(:recruitment_cycle, :next) }
-  let(:provider3) { create(:provider, provider_name: provider2.provider_name, organisations: [organisation], recruitment_cycle: next_recruitment_cycle) }
-  let(:provider4) { create(:provider, organisations: [organisation], recruitment_cycle: next_recruitment_cycle) }
   let(:user) { create :user, organisations: [organisation] }
   let(:payload) { { email: user.email } }
   let(:token) { build_jwt :apiv2, payload: payload }
   let(:credentials) do
     ActionController::HttpAuthentication::Token.encode_credentials(token)
   end
 
-  before do
-    provider1
-    provider2
-    provider3
-    provider4
-  end
```

Comment:
Why are we removing these? It looks like we're removing the tests that certain providers are _not_ in the results, is that intentional?

Reply:
Yep - I think it's about getting the balance right. 
We don't need to create these providers for every spec, so now we are are just creating when required